ui/src/bakery.ts

```diff
@@ -1,7 +1,7 @@
 import { Bakery, BakeryStorage } from "@canonical/macaroon-bakery";
 
 import { actions as statusActions } from "app/store/status";
-import { store } from "index";
+import { store } from "redux-store";
```

Comment:
This was slowing down tests that used http sagas as the http module imported from this file which imported from the index which set up the full app.